js/Particles.js

```diff
@@ -8,7 +8,8 @@ PHOTONS.RangeType = Object.freeze( {
 
 	Cube: 1,
 	Sphere: 2,
-	Plane: 3
+	Plane: 3,
+	Integer: 4

```

Comment:
I think "Integer" as a range type doesn't quite make sense. Range types should describe the "shape" of the area in which the random value could fall. I know you're trying to generate a random value that's clamped to the nearest integer, but we should probably think of a better place to do the rounding. You have identified that we do need a new range type that is one-dimensional though, maybe we should call it something like "Line" or "Default" (I can't think of a good name right now :) ). As for forcing rounding down to the nearest integer, maybe we should make that another parameter for the getRandomXYZ() functions.